src/interface/api.yaml

```diff
@@ -376,6 +376,65 @@ paths:
           $ref: '#/paths/~1health/get/responses/501'
         '503':
           $ref: '#/paths/~1health/get/responses/503'
+  '/thirdpartyRequests/transactions/{ID}/authorizations':
+    post:
+      tags:
+        - thirdpartyRequests
+        - sampled
+      operationId: VerifyThirdPartyAuthorization
+      summary: VerifyThirdPartyAuthorization
+      description: |
+        The HTTP request `POST /thirdpartyRequests/transactions/{id}/authorizations` is used by the DFSP to verify a third party authorization.
+      parameters:
+        - name: ID
+          in: path
+          required: true
+          schema:
+            type: string
+          description: The identifier value.
+        - $ref: '#/paths/~1thirdpartyRequests~1transactions/post/parameters/0'
```

Comment:
question: what's with this ref and other ref's throughout? think I might have missed/forgotten the explanation of this.

Reply:
@kleyow for openapi v3 if you use only $ref to some other definition and if you also define other properties (which already are present in referenced definition then we got a schema validation error -> we faced it for `description` property and because in some situation it is good to give additional description we use a workaround with `allOf` which allows redefinition of existing properties